src/riot-rs-embassy/src/lib.rs

```diff
@@ -95,20 +95,7 @@ async fn init_task(mut peripherals: arch::OptionalPeripherals) {
     println!("riot-rs-embassy::init_task()");
 
     #[cfg(feature = "hwrng")]
-    {
-        // The union of all contexts that wind up in a construct_rng should be synchronized with
```

Comment:
That comment was not moved – I'd consider that valuable because until we have checks to sync those, the mutual references will help keep them in sync.